ee/ucp/admin/install/plan-installation.md

```diff
@@ -42,19 +42,64 @@ this.
 
 ## Avoid IP range conflicts
 
+### Engine
+
+There are two IP ranges used by the engine for the `docker0` and `docker_gwbridge` interface:
+ 
+#### docker0
+
+By default, the Docker creates and configures the host system with a network interface called `docker0`, which is an ethernet bridge device. If you don’t specify a different network when starting a container, the container is connected to the bridge and all traffic coming from and going to the container flows over the bridge to the Docker engine, which handles routing on behalf of the container. 
```

Comment:
Should this be "By default, the Docker Engine..."?

Reply:
Fixed that and a couple of other things I found upon taking a closer look.